controllers/articles.controller.js

```diff
@@ -0,0 +1,21 @@
+const { selectArticleById } = require("../models/articles.model")
+
+const getArticleById = (req, res, next) => {
+  const { article_id } = req.params
+
+  if(isNaN(article_id)) {
+    return res.status(400).send({msg: "Invalid article ID"})
+  }
+
+  selectArticleById(article_id).then((article) => {
+    res.status(200).send({article})
+  }).catch((err) => {
+    if(err.status) {
+      res.status(err.status).send({msg: err.msg})
```

Comment:
a good time to start thinking about setting up error handling middleware so that we dont have to keep re-writing this code!